mapengine/worldwind/apk/src/main/java/mil/emp3/worldwind/layer/MilStdSymbolLayer.java

```diff
@@ -4,6 +4,7 @@
 
 import org.cmapi.primitives.GeoPosition;
 import org.cmapi.primitives.IGeoBounds;
+import org.cmapi.primitives.IGeoMilSymbol;
```

Comment:
Is this import necessary?

Reply:
Probably not if nothing else in the file is changing

Reply:
Removed